src/React/DOM/Props.purs

```diff
@@ -407,6 +407,9 @@ useMap = unsafeMkProps "useMap"
 value :: String -> Props
 value = unsafeMkProps "value"
 
+value' :: Array String -> Props
```

Comment:
Can we call this `values` or `valueArray` or something else please?

Reply:
I renamed to `valueArray`. To keep connection with property name (`value`).